indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -223,6 +223,10 @@
     margin-left: 1em;
 }
 
+.reviewing-state {
```

Comment:
We have several ways to display the reviewing state. I would go for nesting where it belongs or simply a more specific name (better).